.github/workflows/CI.yml

```diff
@@ -1,11 +1,11 @@
 name: CI
```

Comment:
@rkurchin Please check the sequence of steps, please. 
I might have placed the snippet in an unsuitable place in the workflow.